skiplang/prelude/ts/src/sk_node.ts

```diff
@@ -61,11 +61,16 @@ class Env implements Environment {
   fetch(url: URL | string) {
     let filename: string | URL;
     const cwd = process.cwd();
-    if (url && url instanceof URL && url.pathname) {
+    if (typeof url === "string") {
+      filename = url;
+    } else if (url && url instanceof URL && url.pathname) {
       filename = "./" + path.relative(cwd, url.pathname);
-      // @ts-ignore
-    } else if (url && url.default) {
-      // @ts-ignore
+    } else if (
+      url &&
+      // @ts-expect-error: Property 'default' does not exist on type 'URL'.
+      url.default
```

Comment:
It seems that `default` isn't in `string` or `URL`, so I don't know when this case is ever used.

Reply:
It was apparently introduced in [this diff](https://github.com/SkipLabs/skip/commit/8867b216da79c46b254746dc9b10b20a6be944c1#diff-7eb38b23d135400fced06531156114887376359756631d803d3ece6e79e2cabeR70). @skiplabsdaniel any reasons?